delft3dworker/test_models.py

```diff
@@ -339,16 +342,17 @@ def test_export_export(self):
         self.assertEqual(len(zf.namelist()), 1)
 
     @patch('delft3dworker.tasks.chainedtask.delay', autospec=True)
-    def test_start_scene(self, mocked_task_delay):
-        # mocked_task_delay.return_value = {"info": {}, 'id': 22, 'state': ""}
-        mocked_task_delay.return_value.task_id = 22
-        mocked_task_delay.return_value.state = "PROCESSING"
+    def test_start_scene(self, mocked_chainedtask):
+        # mockchainedtask.return_value = {"info": {}, 'id': '22', 'state': ""}
+        mocked_chainedtask.return_value.task_id = '22'
+        mocked_chainedtask.return_value.state = "PROCESSING"
+
         started = self.scene.start()
 
     @patch('delft3dworker.models.revoke_task', autospec=True)
     @patch('celery.contrib.abortable.AbortableAsyncResult', autospec=True)
-    def test_stop_scene(self, mocked_task_delay, mockedResult):
-        mocked_task_delay.return_value = {"info": {}, 'id': 22, 'state': ""}
+    def test_stop_scene(self, mocked_task_delay, mocked_abortable_result):
```

Comment:
mocked_abortable_result : Fedor suggereerde in eerdere sprint reviews, bla bla bla...

https://www.python.org/dev/peps/pep-0008/#class-names
"Class names should normally use the CapWords convention."